source/adminguide/reliability.rst

```diff
@@ -78,25 +156,8 @@ HA features work with iSCSI or NFS primary storage. HA with local
 storage is not supported.
 
 
-HA for Hosts
-------------
-
-The user can specify a virtual machine as HA-enabled. By default, all
```

Comment:
@Spaceman1984 any particular reason to remove this text here? Is this doc moved elsewhere?

Reply:
The title says HA for Hosts, but the content that followed was a duplication of the previous section.